### ISSUE TITLE
Does not seem to work for 2 or more encrypted drives

### ISSUE THREAD

**Issue by githubbiswb** [jikamens/keyless-entry#8]
Love the script, and it works flawlessly for 1 encrypted drive. But when I have it set for 2, my system decrypts the first drive successfully and then throws errors about being unable to find the file and fails to emergency mode. I can then disable it and reboot, get prompted for the passphrases and everything comes up. 

Some details that might be important:
OS: Ubuntu 22
Passphrases for both drives match
Both drives are ext4
When I boot up and keyless-entry is disabled I type in the passprhase at two very different places during the startup process
One of the drives is the boot drive, and that works just fine even in emergency mode, so it is successfully decrypting

Let me know if I can provide you any more details, and again, thank you for the great script

**Comment by jikamens** [comment]
Thanks for the bug report. I will investigate as time permits, but I want to be transparent that I'm swamped in general nowadays so I don't know when I'm going to be able to look at this.

**Comment by githubbiswb** [comment]
Makes sense, and totally understand, again thank you for the great script!
